src/lib/rest/RestService.cpp

```diff
@@ -237,7 +237,7 @@ std::string tenantCheck(const std::string& tenant)
     char numV2[STRING_SIZE_FOR_LONG];
 
     snprintf(numV1, sizeof(numV1), "%d",  SERVICE_NAME_MAX_LEN);
-    snprintf(numV2, sizeof(numV2), "%lu", strlen(name));
+    snprintf(numV2, sizeof(numV2), "%lu", static_cast<unsigned long>(strlen(name)));
```

Comment:
My idea is to some day (hopefully soon) change the C++ compiler for a C compiler, and thus *prohibit* any kind of C++ garbage (sorry, I'm not a fan of C++).
Could not a normal C typecast to e.g. "uint64_t" be done here instead of this "static_cast" thing (that I don't know exactly what it does, but that will fail when I change the compiler for a C compiler)

Reply:
Well, I'm no C++ programmer, I just tried to follow what seems to be "usual" in the C++ universe. I'll check with one of our C++ experts.

Reply:
Yeah, I understand. Never mind C++, just do what you'd do if the compiler was C and not C++

Reply:
Instead of casting, why not use the correct modifier for size_t/ssize_t:
```suggestion
    snprintf(numV2, sizeof(numV2), "%zu", strlen(name));
```

Reply:
ok, sounds good to me. Very good actually
Never used %z ... :)
I was aboiut to propose to simply remove this extra piece of information in the log message. Not really needed.
If %z fixes the problem, let's do that!

Reply:
I'll fix the series.